cpp/docs/DEVELOPER_GUIDE.md

```diff
@@ -747,25 +747,20 @@ void isolated_helper_function(...);
 
 # Deprecating and Removing Code
 
-libcudf is constantly growing and improving to improve performance and better meet our users' needs.
+libcudf is constantly evolving to improve performance and better meet our users' needs.
 As a result, we occasionally need to break or entirely remove APIs to respond to new and improved
 understanding of the functionality we provide. Remaining free to do this is essential to making
 libcudf an agile library that can rapidly accommodate our users needs. We encourage users to
 [Live At Head](https://www.youtube.com/watch?v=tISy7EJQPzI). On a best effort basis, the libcudf
 team will notify users of changes that we expect to have significant or widespread effects.
 
-Where possible, pending API removals will be indicated using the
+Where possible, indicate pending API removals using the
 [deprecated](https://en.cppreference.com/w/cpp/language/attributes/deprecated) attribute and
-documented using Doxygen's [deprecated](https://www.doxygen.nl/manual/commands.html#cmddeprecated)
-command prior to removal. When an API is deprecated, if a replacement exists the replacement should
-be mentioned in both the deprecation message and the deprecation documentation. The PRs introducing
-deprecations should also be labeled "deprecation" to facilitate discovery and removal in the
-subsequent release.
-
-In order to advertise breaking changes, any PR that breaks or removes an existing API should be
-tagged as "breaking". This will ensure that the "Breaking" section of the release notes will include
-a description of what has broken from the past release. Note that PRs containing deprecations should
-be labeled as "non-breaking".
+document them using Doxygen's [deprecated](https://www.doxygen.nl/manual/commands.html#cmddeprecated)
```

Comment:
Line length. I know this is annoying and manual, but the rest of the doc is wrapped at the same column. Wish clang-tidy would do markdown files. This doesn't mention anything about line length: https://github.com/notslang/tidy-markdown

Reply:
I think you looked at the specific commit where I pushed your changes and missed the commit afterwards where reformatted all line lengths. Can you take a look at the current version and make sure the line lengths look like you'd expect?